hi-ml-multimodal/src/health_multimodal/image/inference_engine.py

```diff
@@ -67,3 +68,19 @@ def get_patch_embeddings_from_image(self, image_path: Path) -> Tuple[torch.Tenso
         assert projected_img_emb.shape[0] == 1
 
         return projected_img_emb[0], img_shape
+
+    @torch.no_grad()
+    def get_projected_global_embedding_from_image(self, image_path: Path) -> torch.Tensor:
```

Comment:
I would remove `_from_image` here, as it can be inferred from the argument name and the docstring. Although I know it's common in our codebase, so feel free to ignore this comment.